lib/resources/r4/appointment_patch.yaml

```diff
@@ -132,6 +135,7 @@ operations:
         <li>This patch operation may be used only for Video Visit appointments</li>
         <li>The value for this patch operation will be applied to both the provider Video Visit link and the patient Video Visit link</li>
         <li>This patch operation must be accompanied by all other Video Visit link patch operations with an op of add</li>
+        <li>The URL length must be less than or equal to 255 characters</li>
```

Comment:
same here, not needed.

Reply:
Removed from incorrect location https://github.com/cerner/fhir.cerner.com/pull/462/commits/c4ea6c59054c524481ffaa7ae7e46644ac6eec37